datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/GmsGraphQLEngine.java

```diff
@@ -288,6 +321,13 @@ private static void configureDashboardResolvers(final RuntimeWiring.Builder buil
      * Configures resolvers responsible for resolving the {@link com.linkedin.datahub.graphql.generated.Chart} type.
      */
     private static void configureChartResolvers(final RuntimeWiring.Builder builder) {
+        builder.type("Chart", typeWiring -> typeWiring
+                .dataFetcher("genericDownstreamLineage", new AuthenticatedResolver<>(
```

Comment:
So first, we have to resolve the genericDownstreamLineage which returns a "GenericLineageRelationship". Then we need to resolve the entity within that!

Reply:
Not sure what this comment is saying?